### ISSUE TITLE
No image normalization even with pretrained weight?

### ISSUE THREAD

**Issue by twmht** [openpifpaf/openpifpaf#483]
Hi,

In some of the basenetworks like mobilenet or resnet, you use the pretrained weight from torchvision.

But I can't see any image normalization (like subtracting mean) in preprocess (https://github.com/openpifpaf/openpifpaf/blob/main/openpifpaf/plugins/coco/cocokp.py#L190), in torchvision pretrained models,  they use imagenet mean and variance to normalize the input images (https://github.com/pytorch/examples/blob/97304e232807082c2e7b54c597615dc0ad8f6173/imagenet/main.py#L197).

Any idea?

**Comment by twmht** [comment]
ok. I found that.
